lib/samlp.js

```diff
@@ -131,12 +133,13 @@ function buildSamlResponse(options) {
 
     var pem = encoders.removeHeaders(options.cert);
     sig.keyInfoProvider = {
-      getKeyInfo: function () {
-        return "<X509Data><X509Certificate>" + pem + "</X509Certificate></X509Data>";
+      getKeyInfo: function (key, prefix) {
+        prefix = prefix ? prefix + ':' : prefix;
```

Comment:
`prefix` (function argument) at this point will be a string, empty or with data, right? (just to see if I followed it correctly)